guides/schema/mutations.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Schema — Mutations
+---
+
+## Mutations
+
+Registering a mutation root allows to define fields that can mutate your data.
+
+```ruby
+Schema = GraphQL::Schema.define do
+  query QueryRoot
+  mutation MutationRoot
+end
+
+MutationRoot = GraphQL::ObjectType.define do
+  name "Mutation"
+
+  field :addPost, Post do
+    description "Adds a Post."
+
+    # Use Input Types to define complex argument types
+    argument :post, PostInputType
+    resolve ->(t, args, c) {
+      title = args['post']['title']
+      description = args['post']['description']
+      Post.create(title: title, description: description)
+    }
+  end
+end
+
+PostInputType = GraphQL::InputObjectType.define do
+  name "PostInputType"
+  description "Properties for creating a Post"
+
+  argument :title, !types.String do
+    description "Title of the post."
+  end
+
+  argument :description, types.String do
+    description "Description of the post."
+  end
+end
+```
+
+## Nested input types
```

Comment:
Have you considered making a nested mutation that creates multiple post items? The example above already defines the type so it might be easier for the reader to continue with this example set.